dabl-meta.yaml

```diff
@@ -4,7 +4,7 @@
 catalog:
     name: dabl-integration-core
     group_id: com.digitalasset
-    version: 0.9.2
+    version: 0.9.9
```

Comment:
Once this has been in service for a bit, will call it `1.0.0`, and start thinking about next steps.